test/subgraph-tests/position.test.ts

```
@@ -28,12 +28,28 @@
     console.log('Position data:::', positionData);
     expect(positionData.length).toEqual(2);
   });
+  it('should return only open positions of this timeframe', async () => {
+    const parifiSdk = await getParifiSdkInstanceForTesting();
+    const startTime = 1735689600;
+    const endTime = 2035689600;
+    const positionData = await parifiSdk.subgraph.getAllOpenPositionsWithTime(startTime, endTime);
+    console.log('Position data:::', positionData);
+    expect(positionData.length).toEqual(6);
+  });
+  it.only('should return only closed positions of this timeframe', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)